src/blocks/textInput/TextInput.tsx

```diff
@@ -13,7 +13,7 @@ export type TextInputProps = {
   type?: 'text' | 'password';
   errorMessage?: string;
   label?: string;
-  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Why make it optional?

Reply:
Because even when the input is disabled like in the  last section - verify alias, Text input shows error